### ISSUE TITLE
performance testing?

### ISSUE THREAD

**Issue by mgenev** [dai-shi/use-context-selector#10]
Hi,
this package is very clean and I'd very much like to use it. Have you done performance testing on it?

**Comment by dai-shi** [comment]
Hi!
I haven't done performance testing specifically to this library,
but the same technique is used in both [react-tracked](https://github.com/dai-shi/react-tracked) and [reactive-react-redux](https://github.com/dai-shi/reactive-react-redux).
I've done lots of testing there.

The latest one is shown at https://github.com/dai-shi/react-tracked#benchmarks

Let me know if you have any question reading the table.
In short, it's as performant as you'd expect.

**Comment by mgenev** [comment]
Thank you! We are comfortable using the module then.
